### DISCUSSION TITLE
Dropdown problem

### DISCUSSION THREAD

**Discussion by fralvareza** [primefaces/community#1433] [PrimeNG Templates]
Hi,

I'm working on a new app using the Poseidon template, now v17, but now i have a problem using dropdown, when get the data and trying to display in a dropdown component is not working, but if i show the form empty the dropdown appear without problem.

Is there problem with this component?

Thanks.

Frederic

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Reply by fralvareza** [reply]
Please, see the file attached, If you see the dropdown component is not show normal, especially when the data is populate it

![primeng_dropdown](https://github.com/primefaces/community/assets/20777902/98825624-0078-4b87-8ec6-997a8eb953bb)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example by forking this example https://stackblitz.com/edit/hyyv8h?file=src%2Fapp%2Fdemo%2Fdropdown-basic-demo.ts

Thank you for sharing the image but it doesn't help in this case. We need to see code example to investigate and identify the issue.
